buildah/buildah.yaml

```diff
@@ -7,7 +7,7 @@ spec:
     params:
     - name: BUILDER_IMAGE
       description: The location of the buildah builder image.
-      default: quay.io/buildah/stable
+      default: quay.io/buildah/stable:v1.11.3
```

Comment:
@danielhelfand could you change it to `v1.11.0 `?

Reply:
Pushed it up. Thanks for clarifying the version.